verifyFiles.py

```diff
@@ -193,13 +198,38 @@ def pretty_string(diff_dict,conditions_dict):
         output_string+="\n"
     return output_string
 
+#Method is_subject_folders_same checks if the subject_folders under different conditions are the same. If not , it stops the execution of the script.
+def is_subject_folders_same(conditions_list,root_dir):
+   if len(conditions_list) > 1:
```

Comment:
To avoid indenting the whole method body, you could just return True if there is one or less conditions.